website_form_recaptcha/views/website_config_settings.xml

```diff
@@ -1,31 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
     Copyright 2018 Simone Orsi - Camptocamp SA
+    Copyright 2019 Tecnativa - Cristina Martin R.
     License AGPL-3 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 <odoo>
 
     <record id="view_website_config_settings" model="ir.ui.view">
+        <field name="neme">reCaptcha Settings</field>
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="website.res_config_settings_view_form" />
```

Comment:
website_form.res_config_settings_view_form

Reply:
Thank you for your review @mreficent , but I think it's better display it on global view, in the communication section, next to configuration forms. Or is it for another reason?

Reply:
This module depends of `website_form`, so if `website_form` has a view, then I think you should here inherit from this view instead to avoid possible conflicts between views. You can be calm, it doesn't affect the rest of the code. The things you display in this view will still be displayed in the same place.

Reply:
But the arch in which this inherited view relies is in `website_form` or in `website`?

Reply:
Is it relevant?

Reply:
Yes, you always have to try to inherit from the top view for not depending on useless inheritances.

Reply:
But here the top view is which? The one directly affected by the arch (website) or the one that depends directly by the module dependencies (website_form)?

Reply:
Hey, I remind that this is a non-blocking "comment", not a "request changes". If you consider that it's better not do "useless inheritances", then don't do it.

Reply:
I know, but as always, I like to have with you a sane discussion about the topic for being instructive for all. Imagine this simplified archs:
id='module_a.view1'
```xml
<form>
  <field name="X"/>
</form>
```
id='module_b.view1' inherit_id='module_a.view1'
```xml
<field name="X" position="after">
  <field name="Y"/>
</form>
```
if you need to make a new view for placing something after field X, you don't need to inherit from `module_b.view1`, but from `module_a.view1`, as the field X is already defined there. If you need to place your field referencing field Y, then you must inherit indeed from module_b, and this is independent if you need to depend your module in module_b in the manifest due to other thing.

Do you understand me?

Reply:
If module_b.view1 is the following:
```xml
<field name="X" position="replace">
</form>
```
or similar with `position="move"`. Then, you may have problems.

Reply:
Well, then the problem is because module_b is bad designed. See my talk about module compatibility, which basically forbids the `replace` operation if you want a "compatible module". But if not, `module_b.view1` priority should be set to a very low priority for being processed the last one. That way, there's no problem.

Reply:
But I think you can't avoid bad designed modules or people using `replace` 🤔

Reply:
Buy hey, you got my point and I got your point. @cristinamartinrod feel free  😁

Reply:
Not really, because there can always be other views outside your radar that makes the same, so, always inherit on the lower view that you need, and don't put in your system bought apps from Apps store :stuck_out_tongue_closed_eyes: